js/content/store.js

```diff
@@ -1493,6 +1493,12 @@ class AppPageClass extends StorePageClass {
             linkNode.href = `https://store.steampowered.com/search/?${name}=${encodeURIComponent(value)}`;
             HTML.afterEnd(linkNode, ` (<a href="${homepageLink.href}">${Localization.str.options.homepage}</a>)`);
         }
+
+        for (let moreBtn of document.querySelectorAll(".dev_row > .more_btn")) {
+            if (moreBtn) { moreBtn.remove(); }
```

Comment:
if that if statement necessary?

Reply:
yeah, otherwise the button gets duplicated on columns that already have them

Reply:
Hmm, very well

Reply:
I don't understand too, how can `moreBtn` not be truthy?

Reply:
I assume `document.querySelectorAll(".dev_row > .more_btn")` may select an element more than once and it will not exist anymore after the initial `remove()`.

Reply:
Steam adds the button on page load, so some columns already have them. After we add homepage links, some columns that didn't need the button might need it, so we run the script again to add it, but for the columns already have the button, this will dupe it because the column style isn't changed, so I thought to remove all buttons first.

Reply:
I see that, but the question I wanted to get an answer for was:
How can `if (moreBtn)` ever be `false`?

Anyway, I will remove this `if` statement